src/fmt.zig

```diff
@@ -75,6 +75,18 @@ fn formatStatement(fmt: *Formatter, si: StatementIdx) NewlineBehavior {
         .import => |i| {
             fmt.buffer.appendSlice(fmt.gpa, "import ") catch |err| exitOnOom(err);
             fmt.formatIdent(i.module_name_tok, i.qualifier_tok);
+            if (i.exposes.span.len > 0) {
+                fmt.pushAll(" exposing [");
+                var n: usize = 0;
+                for (fmt.ast.store.tokenSlice(i.exposes)) |tok| {
+                    fmt.pushTokenText(tok);
+                    if (n < i.exposes.span.len - 1) {
+                        fmt.pushAll(", ");
+                    }
+                    n += 1;
+                }
+                fmt.push(']');
```

Comment:
Will there be a coherent effort at some point to handle optional multiline formatting (either based on trailing commas or newlines)? I presume so, which is why I haven't been commenting about it.

Reply:
We also need to support moving comments over, and my thinking is to handle those both in the same way. (altho I do like the trailing comment behavior!)

Reply:
Yes, all of that is coming

Reply:
I plan to do it in one small place first, and then the rest of the places later